Build-Core.cmd

```diff
@@ -2,7 +2,7 @@
 setlocal
 
 SET CMDHOME=%~dp0.
-SET BUILD_FLAGS=/m:1 /v:m /fl /flp:logfile=build.log;verbosity=detailed
```

Comment:
As we talked in person, binary log and console output (the latter at `normal` verbosity) should be good for our needs